modules/Oculus_module/src/OculusModule.cpp

```diff
@@ -23,6 +23,29 @@
 
 #include <functional>
 
+Eigen::Ref<Eigen::Matrix<double, 3, 3, Eigen::RowMajor>>
+getRotation(yarp::sig::Matrix& m)
+{
+    assert(m.rows() == 4 && m.cols() = 4 && "getRotation expects a 4x4 matrix");
+    return Eigen::Map<Eigen::Matrix<double, 4, 4, Eigen::RowMajor>>(m.data()).topLeftCorner<3,3>();
+}
+
+auto getPosition(yarp::sig::Matrix& m)
```

Comment:
I am not sure if this is ``c++14``.

Reply:
😭  I'm not able to make the code working without auto. I should return the type of 
```
Eigen::Map<Eigen::Matrix<double, 4, 4, Eigen::RowMajor>>(m.data()).topRightCorner<3,1>()
```